SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -452,6 +470,76 @@ void CTurbSSTSolver::BC_HeatFlux_Wall(CGeometry *geometry, CSolver **solver_cont
   }
 }
 
+
+void CTurbSSTSolver::SetNuTilde_WF(CGeometry *geometry, CSolver **solver_container, CNumerics *conv_numerics,
+                                  CNumerics *visc_numerics, CConfig *config, unsigned short val_marker) {
+
+  const unsigned short max_iter = 100;
+
+  /* --- tolerance has LARGE impact on convergence, do not increase this value! --- */
+  const su2double tol = 1e-12;
+
+  /*--- Typical constants from boundary layer theory ---*/
+
+  const su2double kappa = 0.4;
```

Comment:
Aren't some of the constants defined at solver level?

Reply:
True, not this one though. It is however also defined in the turbulence solvers with a slightly different value (0.40 vs 0.41, but in the literature you also find 5.0) I think the best is to make this a config option with a good default value, and people can change it if they really want to. In any case such magic numbers should be collected at a central location and not be hidden. It is even less hidden if it is a config option.